src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -115,6 +115,20 @@ def get_action(self, observation):
                     dict(mean=dist.mean.squeeze(0).numpy(),
                          log_std=(dist.variance**.5).log().squeeze(0).numpy()))
 
+    def forward(self, *inputs):
```

Comment:
please document the *actual* inputs, as individual arguments, with descriptive names (e.g. observations).